tests/rt.conf

```diff
@@ -189,12 +189,14 @@ RUN     | datm_cdeps_debug_cfsr
 # ATM-WAV tests                                                                                                                                                                   #
 ###################################################################################################################################################################################
 
-COMPILE | -DAPP=ATMW -DCCPP_SUITES=FV3_GFS_v16                                                                                  | - wcoss_cray  | fv3 |
-RUN     | control_atmwav                                                                                                        | - wcoss_cray  | fv3 |
+COMPILE | APP=ATMW SUITES=FV3_GFS_v16 32BIT=Y                                                                                   | - wcoss_cray                          | fv3 |
```

Comment:
@aliabdolali this line needs to be reverted to what it is in the develop branch
as is the model does not compile (which is why I still don't have updates on gaea)

Reply:
@JessicaMeixner-NOAA I reverted it. However, 32BIT=Y  was suggested by @DeniseWorthen

Reply:
Adding 32BIT=Y is a great suggestion, but the "APP=ATMW and SUITES=.." is no longer the correct format and the model will not build as it was.   Are we going to use 32bit?   If so, I need to know so I can properly update the test for gaea.

Reply:
I think it is good to have 32BIT=Y as this test is a low resolution test for GFSv16.

Reply:
I added 32BIT=Y to it.

Reply:
Does it compile?  The other compile lines with 32BIT use "-D32BIT=ON"?

Reply:
Things are changing rapidly. It was working a few weeks ago. I changed it to -D32BIT=ON and tested it.

Reply:
Just trying to help out, since gaea is down and because I was trying to debug the compiler errors I was getting on gaea, I have run on hera as well.  The results are here: /scratch1/NCEPDEV/stmp2/Jessica.Meixner/FV3_RT/rt_203871 @aliabdolali can you confirm this in line with your results from the last time you ran and had things working on hera for the two ATMW tests?

Reply:
Thanks Jessica, I checked the log files, run times and the list of wave out_grd and out_pnt and restart files. They look ok. I also ran the whole rt.conf last night on hera, as I synced my branch with the ufs-weather-model and all ran successfully.

Reply:
So it's expected that in both cases that the atm model runs about 2x as fast as the wave model?  This is what I'm seeing in the ESMF_Profile.summary, just want to confirm.

Reply:
From a conversation we had with @junwang-noaa, we do not want to allocate more cpu than this to the wave model. otherwise, we might be able to reduce the whole simulation by adding more cores. Or I can reduced the atm cores and add them to the wave model. But again, Jun mentioned in the past we had 1/3 to wave, 2/3 to atm.